solc/CommandLineParser.cpp

```diff
@@ -783,7 +780,7 @@ General Information)").c_str(),
 	optimizerOptions.add_options()
 		(
 			g_strOptimize.c_str(),
-			"Enable bytecode optimizer."
+			"Enable optimizer."
```

Comment:
This is the EVM assembly optimizer, right? Maybe it would be nice to also add this, e.g. `Enable optimizer. (EVM assembly optimizer)`

Reply:
No, `--optimize` is all of the optimizer. Using this option alone enables all optimizer components: EVMASM, Yul and potentially any other component we might add in the future.

There's no option to control the EVM assembly optimizer in isolation. However, in the current situation where we basically have only two components, you can do that indirectly by controlling the Yul optimizer: `--optimize --no-yul-optimizer` will effectively give you only EVMASM optimization and `--no-yul-optimizer` will give you all optimization except for EVMASM.

Reply:
Ah true! Yeah, if we do that CLI redesign, we should probably also rethink all the options and parameters. It's not very intuitive.